src/plugins/theme/maxWidth.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Max Widths - customized to not produce classes with our custom spacing values
+ */
+module.exports = function () {
+  return {
+    none: 'none',
+    0: '0',
+    10: '10rem',
+    20: '20rem',
+    30: '30rem',
+    40: '40rem',
+    50: '50rem',
+    60: '60rem',
+    70: '70rem',
+    80: '80rem',
+    90: '90rem',
+    100: '100rem',
+    full: '100%',
+    min: 'min-content',
+    max: 'max-content',
+    prose: '65ch',
```

Comment:
overriding the TW defaults because the default generated a bunch of classes using our spacing steps (which in our cases don't really make sense as max-widths.